OrbitClientServices/CrashManager.cpp

```diff
@@ -40,9 +40,7 @@ void CrashManagerImpl::CrashOrbitService(
   grpc::Status status =
       crash_service_->CrashOrbitService(&context, request, &response);
 
-  if (!status.ok()) {
-    ERROR("Grpc call failed: %s", status.error_message());
-  }
+  CHECK(status.error_code() == grpc::StatusCode::DEADLINE_EXCEEDED);
```

Comment:
maybe have an if() with log - or even dialog if error_code is not DEADLINE_EXCEEDED. CHECK seems like an overkill here.

Reply:
Added an error to the log